Sources/CSRFFormFieldTag.swift

```diff
@@ -0,0 +1,20 @@
+import Vapor
+
+public final class CSRFFormFieldTag: TagRenderer {
+    public init() {}
```

Comment:
Do you need this empty `init`?

Reply:
Well, to be honest, I don't know. I just copied the example of the docs. And it's done in the official tags: https://github.com/vapor/template-kit/blob/1.1.0/Sources/TemplateKit/Tag/Lowercase.swift

But I just tested and it seems to work without the empty init. To be consistent with the existing tags I'd just leave it there..?!

Reply:
I agree, but I think I prefer consistency, so I'd leave it like this.